.circleci/config.yml

```diff
@@ -1,47 +1,9 @@
 version: 2.1
 orbs:
   gcp-gcr: circleci/gcp-gcr@0.15.0
-jobs:
-  nix-mac:
```

Comment:
This job was commented out down below.  I'm not sure if it didn't work, or if we decided it was redundant, but I'm going ahead and removing it for the time being.  We can always restore it later by looking back in the git history.